content/learn/beginner/control-flow-1.md

```diff
@@ -126,6 +126,23 @@ if you try to re-define a variable using `:=`, but it will not affect the outer
 
 {{< /callout >}}
 
+<div style="float: right; margin-left: 7px;">
+
+![Responses: template: :XX: unexpected EOF](unexpected-eof.png)
+
+</div>
+
+Try and recall the last example from the introduction of if-statements and notice how, despite us having multiple blocks
+of branching code, there's still only one `{{ end }}` action.
```

Comment:
I'm not too sure how this sentence is relevant; could you clarify?

Reply:
```
if {
    // code
} else {
    // other code
}
```
Note the additional `{}` for the else block; if we're going to say "yeah this is like the closing curly brace", this "but actually only once" seems relevant to me. Not too sure how to phrase this better, or if it's just better to outright remove it.
But then, I do have seen code like this (please do not ask me when / where):

```
{{ if pipeline }}
    code
{{ end }}
{{ else }}
    more code
{{ end }}
```